### PR TITLE
feat(CX-3229): separate between past and future auction results

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [CX-3229] 

### Description

This PR separates between past and future auction results.

https://user-images.githubusercontent.com/11945712/208971597-10e9e046-ea83-4bc0-baa8-b1e6a831281a.mov

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [x] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- separate between past and future auction results - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3229]: https://artsyproduct.atlassian.net/browse/CX-3229?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dariakoko — DISMISSED** [review]
Great work!

**Review by MrSltun — DISMISSED** [review]
💫

File: src/app/Components/Artist/ArtistInsights/ArtistInsightsAuctionResults.tests.tsx [hunk 1]
```diff
@@ -69,19 +70,92 @@ describe("ArtistInsightsAuctionResults", () => {
...
   )
 
-  it("renders list auction results when auction results are available", () => {
-    const tree = renderWithWrappersLEGACY(<TestRenderer initialData={initialState} />)
-    resolveMostRecentRelayOperation(mockEnvironment, {
-      Artist: () => ({
-        auctionResultsConnection: {
-          totalCount: 5,
-          edges: mockEdges(5),
+  describe("Upcoming auction reuslt", () => {
```
Comment (lordkiz) [inline src/app/Components/Artist/ArtistInsights/ArtistInsightsAuctionResults.tests.tsx:73]:
```suggestion
  describe("Upcoming auction results", () => {
```

File: src/app/Components/Artist/ArtistInsights/ArtistInsightsAuctionResults.tests.tsx [hunk 2]
```diff
@@ -69,19 +70,92 @@ describe("ArtistInsightsAuctionResults", () => {
...
+          return {
+            totalCount: 5,
+            edges: mockEdges(5),
+          }
+        },
+      })
+
+      expect(tree.root.findAllByType(SectionList).length).toEqual(1)
+      expect(extractText(tree.root.findByType(SectionList))).not.toContain("Past Auctions")
+    })
```
Comment (lordkiz) [inline src/app/Components/Artist/ArtistInsights/ArtistInsightsAuctionResults.tests.tsx:158]: Nice tests. Clean, very readable descriptions. 🙌

Comment (MounirDhahri) [reply src/app/Components/Artist/ArtistInsights/ArtistInsightsAuctionResults.tests.tsx:158]: Thank you 🙏

**Review by lordkiz — APPROVED** [review]

**Pull request merged**